skt/executable.py

```diff
@@ -220,68 +235,66 @@ def cmd_merge(cfg):
                                        merge_report_path, report_string)
 
                     return
-
                 report_string += '\n  - %s\n    into commit %s' % (mb[0],
                                                                    bhead[:12])
-                idx += 1
-
-        elif cfg.get('patch') or cfg.get('pw'):
-            report_string = 'We applied the following patch(es):'
-
-            if cfg.get('patch'):
-                for patch in cfg.get('patch'):
+                previous_merge = thing_to_merge
+                idx[0] += 1
+
+            elif thing_to_merge[0] == 'patch' or thing_to_merge[0] == 'pw':
```

Comment:
There's no other option that can be applied besides `merge_ref`, `patch` and `pw`. If should be enough to use a siple `else` instead of the condition here.

Reply:
yeah it is better